apps/condo/pages/ticket/index.tsx

```diff
@@ -49,7 +49,7 @@ const TicketsPage: IPageWithHeaderAction = () => {
     const CreateTicket = intl.formatMessage({ id: 'CreateTicket' })
     const EmergencyLabel = intl.formatMessage({ id: 'Emergency' })
     const DownloadExcelLabel = intl.formatMessage({ id: 'pages.condo.ticket.id.DownloadExcelLabel' })
-
+    const timeZone = intl.formatters.getDateTimeFormat().resolvedOptions().timeZone
```

Comment:
It's better to use locale from organization

Reply:
Added task SBERDOMA-770